source/changelog.txt

```diff
@@ -12,6 +12,48 @@ Release Notes
    :depth: 1
    :class: singlecol
 
+v2.3.7
+------
+
+*Released December 17, 2024*
+
+Bug Fixes:
+
+- :issue:`MONGOSH-1943` - Only ever run ``MongoshRepl.onExit`` once
```

Comment:
```suggestion
- :issue:`MONGOSH-1943` - Fix cause of rare flaky "Mongosh not initialized yet" errors 
```

This is how this would show up to users, I don't think the raw commit title helps users understand what's going on here